app/javascript/controllers/organization_form_controller.js

```
@@ -164,4 +165,23 @@ export default class extends Controller {
             });
         }
     }
+
+    setupDeleteModal() {
+        const deleteInput = document.getElementById('deleteOrgConfirmInput');
+        const deleteBtn = document.getElementById('deleteOrgSubmitBtn');
+        if (!deleteInput || !deleteBtn) return;
+
+        const orgName = deleteInput.dataset.orgName;
```

ESLint

[error] Use object destructuring.

(prefer-destructuring)